components/ProfileCard.vue

```diff
@@ -0,0 +1,55 @@
+<script setup>
+
+	const props = defineProps({
+		firstName: {
+			type: String,
+			required: true,
+		},
+		lastName: {
+			type: String,
+			required: true,
+		},
+		functionTitle: {
+			type: String,
+			required: false,
+		},
+		email: {
+			type: String,
+			required: false,
+		},
+		telephone: {
+			type: String,
+			required: false,
+		},
+		imgUrl: {
```

Comment:
For consistency I'd go for `image`

```suggestion
		image: {
```